test/templates.js

```diff
@@ -347,6 +350,31 @@ describe('script-templates', function () {
     })
   })
 
+  describe('witnessPubKeyHash.input', function () {
+    fixtures.valid.forEach(function (f) {
+      if (f.type !== 'pubkeyhash' && f.type !== 'witnesspubkeyhash') return
+      if (!f.inputStack) return
+
+      var pubKey = Buffer.from(f.pubKey, 'hex')
```

Comment:
use fromHex here?

Reply:
@junderw I was only using `fromHex` in the `.map(fromHex)` case,   if we had ES6 lambdas,  I wouldn't have bothered.